### ISSUE TITLE
Chainsaw doesn't do appropriate damage.

### ISSUE THREAD

**Issue by Coalf** [Aurorastation/Aurora.3#4586]
So the wieldforce of chainsaw is about 40 right? Well the chainsaw does 20 brute not 40. I tested this multiple times in thunderdome using various mobs and weapons, only the chainsaw has this problem.
BUT I noticed the throwforce is 20, so maybe they switched somehow?

I tested it myself, Seb witnessed it.

**Comment by Sindorman** [comment]
No, it seems like whoever coded it though that `force` is `force_unwielded`. I will fix it, also buff wielded force with powered on chainsaw from 40 to 60, cause it is a CHAINSAW!
